src/main/java/com/google/jenkins/plugins/credentials/oauth/GoogleRobotPrivateKeyCredentials.java

```diff
@@ -172,7 +188,7 @@ public static String getHelpFile() {
 
   @Override
   public CredentialsScope getScope() {
-    return CredentialsScope.GLOBAL;
+    return getCredentialsScope();
```

Comment:
Can we just remove `getScope` method and rely on inherited one? I guess `BaseCredentials#getScope` should work for us. It may mean that we don't need `getCredentialsScope` at all and use base class method instead?